### PR TITLE
[Goonchem] New Vending Machines

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Adds in new vending machines and tweaks existing ones

Existing
- Snack vending machines now have Discount Dan's Burritos and Discount Dan's soup
 - Very cheap, but be prepared to deal with the consequences...
 - Soups and burritos both have a self-heating mechanism so you can enjoy nice, tasty, warm food
- Hot drinks vending machine now has chocolate, weight loss drinks, and chicken soup

New
- Adds in the soda vending machine
 - This has premium sodas that cost you more, but the sodas have unique reagents in them. For example, one drink has a tiny amount of meth in it that could help you in a combat situation; others have antihol in them to help you out if you're totally smashed--etc.

The above drinks, burrito, and soup combo opens up a world of opportunity for DIY hobo chemists--especially now that improvised heating is a thing

Food Sprites

burrito
credit to VG

soup
Made this one myself; based off of an existing sprite and heavily modified

Other
- Adds in a few more reagents (alchemical related)
- Adds in Stinkeye's Reserve (we need a sprite for this)
 - For the love of pete, don't drink this shit.

Map
- Adds the new soda vending machine to the map

### CODE REVIEW

**Comment by fj45** [comment]
Pictures of sprites?

**Comment by Midaychi** [comment]
If you somehow micromanaged the heat of the reagents just right, you could inject reagents into the soup or burritos and use it as a timed bomb. You'd just have to figure out a chemical combination that only needs 2 pyrosium worth of heating to go off.

**Comment by CrAzYPiLoT-SS13** [comment]
Stinkeye's Reserve, the sprite, what's it like? Rum?

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -3367,4 +3367,119 @@
...
 	New()
 		..()
-		reagents.add_reagent("ectoplasm", 10)
\ No newline at end of file
+		reagents.add_reagent("ectoplasm", 10)
+
+////Discount Dan's Burritos//////       May God have mercy on your souls---and stomachs.    -Fox
+
+/obj/item/weapon/reagent_containers/food/snacks/danburrito
+	name = "Discoutn Dan's Burrito"
```
Comment (Iamgoofball) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: >discoutn

**Comment by Fox-McCloud** [comment]
fixed the typo.

**Comment by Fox-McCloud** [comment]
@ZomgPonies  @Markolie 

Merge when? D=

**Pull request merged**